app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module SessionsHelper
```

Comment:
Helper files should not have application logic in it according to your requirements that can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from pathwright

Helper methods ought to be view specific; the fact that you have to import the helpers in Application Controller for you to have access to those methods in controllers while you do not even use it in your views reiterates this. The views on the other hand, automatically have access to helper methods. 
So, what you can do is add all methods which are not views specific to your application controller directly. What you can also do if your method list gets long, you may use concerns or a lib folder to separate methods by function, etc.

Basically helpers = views, controller = controller. Why write methods in helpers only to have to include it in controller after. Helpers are automatically accessible in views. there is a reason helpers are outside the context of the controller folder. You can, however, use concerns in your controllers.

Read [this article](http://codefol.io/posts/Where-Do-I-Put-My-Code/)  and [this article](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers) for a clearer explanation. 🙂